### PR TITLE
Improves signaling documentation

### PR DESCRIPTION
**Pull request by Thiago-Dantas**
Improves signal documentation by being more explicit about required configurations

Documents SQL Server quirks

Provides an example for incremental snapshotting

https://issues.redhat.com/browse/DBZ-3699

### CODE REVIEW

**Comment by Thiago-Dantas** [comment]
I'm not 100% sure on this "It will start an incremental snapshot operation up to the oldest available streamed event."

Can someone make sure this correct, or should I remove this entirely?

I think this should be a nice tidbit detail to be known about the feature

**Comment by gunnarmorling** [comment]
@jpechane, could you take a look at this one?

**Comment by gunnarmorling** [comment]
Hey @Thiago-Dantas, thanks a lot! Could you squash the two commits, change the commit message so it begins with "DBZ-3699 Improving..." and then force-push to this PR's branch? Thx!

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 1]
```diff
@@ -127,3 +128,42 @@ If a placeholder `{}` is added to the message it will be replaced with streaming
...
+
+
+.Action parameters
+[cols="3,9",options="header"]
+|===
+|Name | Description
+
+|data-collections
+|The list of data collections to be snapshoted. +
+For MySQL, PostgresSQL and Db2 the collections should be formatted as "some_schema.some_table" +
```
Comment (jpechane) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:150]: This is not exact as for MySQL it is the name of database, not schema.

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 2]
```diff
@@ -127,3 +128,42 @@ If a placeholder `{}` is added to the message it will be replaced with streaming
...
+
+.Action parameters
+[cols="3,9",options="header"]
+|===
+|Name | Description
+
+|data-collections
+|The list of data collections to be snapshoted. +
+For MySQL, PostgresSQL and Db2 the collections should be formatted as "some_schema.some_table" + 
+For SQL Server it should be formatted as "some_database.some_schema.some_table"
```
Comment (jpechane) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:151]: A formating detail "some_database.some_schema.some_table" (and other instances) should be formatted as `"some_database.some_schema.some_table"`

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 3]
```diff
@@ -127,3 +128,42 @@ If a placeholder `{}` is added to the message it will be replaced with streaming
 |`{"message": "Signal message at offset {}"}`
 
 |===
+
+
+=== Triggering an Incremental Snapshot
+*Available for MySQL, PostgresSQL, SQL Server and Db2 connectors*
+
+The type of the action is `execute-snapshot`.
+It will start an incremental snapshot operation up to the oldest available streamed event.
```
Comment (jpechane) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:137]: Unfortunately I am really not sure what you want to express here. Could you please elaborte a bit more?

Comment (Thiago-Dantas) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:137]: I wanted to convey what is the end point of the incremental snapshot operation, this was my best guess lol and was hoping someone could help me on this

File: documentation/modules/ROOT/pages/configuration/signalling.adoc (hunk 1 above)
Comment (Thiago-Dantas) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:150]: I thought for MySQL schema and database are interchangeable, but I will clarify

File: documentation/modules/ROOT/pages/configuration/signalling.adoc (hunk 3 above)
Comment (jpechane) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:137]: I see. So no, this is not accurate. The incremental snapshot finds the largest PK of the table at start of the snapshot of that table and finishes when this point is reached.

Comment (Thiago-Dantas) [reply documentation/modules/ROOT/pages/configuration/signalling.adoc:137]: Will correct, thanks

**Comment by Thiago-Dantas** [comment]
@jpechane applied your suggested corrections

**Comment by Thiago-Dantas** [comment]
@gunnarmorling @jpechane anything missing or should be changed?

**Review by Naros — APPROVED** [review]
LGTM @Thiago-Dantas !

**Pull request merged**